src/lib/components/Swap/Input.tsx

```diff
@@ -39,6 +43,13 @@ export default function Input({ disabled }: InputProps) {
   // extract eagerly in case of reversal
   usePrefetchCurrencyColor(swapInputCurrency)
 
+  const routeIsLoading = useMemo(
```

Comment:
This should be `isLoading` or `isRouteLoading`. Starting a boolean with `is`/`was` makes it a clearer variable name.